sky/adaptors/azure.py

```diff
@@ -74,10 +78,137 @@ def get_client(name: str, subscription_id: str):
         elif name == 'resource':
             from azure.mgmt.resource import ResourceManagementClient
             return ResourceManagementClient(credential, subscription_id)
+        elif name == 'storage':
+            from azure.mgmt.storage import StorageManagementClient
+            return StorageManagementClient(credential, subscription_id)
+        elif name == 'authorization':
+            from azure.mgmt.authorization import AuthorizationManagementClient
+            return AuthorizationManagementClient(credential, subscription_id)
+        elif name == 'graph':
+            from msgraph import GraphServiceClient
+            return GraphServiceClient(credential)
```

Comment:
Import module instead of class:


```suggestion
        elif name == 'graph':
            import msgraph
            return msgraph.GraphServiceClient(credential)
```